lua/lint/linters/vulture.lua

```diff
@@ -1,12 +1,22 @@
 -- path/to/file:line: message
-local pattern = '([^:]+):(%d+):(.*)'
+local pattern = '([^:]+):(%d+): (.*)'
 local groups = { 'file', 'lnum', 'message' }
 
 return {
   cmd = 'vulture',
   stdin = false,
-  args = {},
+  args = {"--exclude='/**/docs/*.py,/**/build/*.py'", function()
+    local output = vim.fn.system("git rev-parse --show-toplevel"):sub(1, -2)
```

Comment:
Why the `:sub(1, -2)` ?

Reply:
Because the command retrieves a string with a special character at the end. This character, when used as input argument to vulture, ended up in the linter not properly working (basically there were no diagnostics). I verified that the current resulting string is the correct one by 1) comparing it with a hardcoded string of the expected path and 2) observing the same diagnostics were produced in the Vulture CLI and in Neovim